packages/swapper/src/constants.ts

```diff
@@ -91,7 +91,6 @@ export const swappers: Record<
 // Slippage defaults. Don't export these to ensure the getDefaultSlippageDecimalPercentageForSwapper helper function is used.
 const DEFAULT_SLIPPAGE_DECIMAL_PERCENTAGE = '0.002' // .2%
 const DEFAULT_COWSWAP_SLIPPAGE_DECIMAL_PERCENTAGE = '0.005' // .5%
-const DEFAULT_PORTALS_SLIPPAGE_DECIMAL_PERCENTAGE = '0.01' // 1%
```

Comment:
👋🏽

Reply:
My assumption was that if we can't get a successful `/v2/estimate` then we probably can't proceed anyway, but I can add it in as another fallback.

Reply:
Addressed in https://github.com/shapeshift/web/pull/7672/commits/102812593a73ebd529202b70c88a65b76f9f0f5c 👌